src/dms/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/UpdateDocumentById.cs

```diff
@@ -64,8 +63,7 @@ Guid[] invalidReferentialIds
     public async Task<UpdateResult> UpdateById(
         IUpdateRequest updateRequest,
         NpgsqlConnection connection,
-        NpgsqlTransaction transaction,
-        TraceId traceId
```

Comment:
Problem where we were passing in traceId again, although already in request object. Led to some test problems.